App.tsx

```diff
@@ -46,7 +47,9 @@ const App: FunctionComponent = () => {
             >
               <PermissionsProvider>
                 <ExposureProvider>
-                  <MainNavigator />
+                  <SystemServicesProvider>
```

Comment:
I might put SystemServices around Exposures as it feels like the exposure logic is dependent on knowing about the system

Reply:
Good call!